File: book-model/src/main/java/org/gdgcd/demo/service/BookMeta.java
package org.gdgcd.demo.service;

import com.fasterxml.jackson.annotation.JsonProperty;

public class BookMeta {
    @JsonProperty("ID")
    public String id;

    @JsonProperty("Title")
    public String title;

    @JsonProperty("SubTitle")
    public String subTitle;

    @JsonProperty("Description")
    public String description;

    @JsonProperty("Image")
    public String imageUrl;
    public String isbn;

}

File: book-model/src/main/java/org/gdgcd/demo/service/BookMeta.kt
package org.gdgcd.demo.service

import com.fasterxml.jackson.annotation.JsonProperty

class BookMeta {
    @JsonProperty("ID") var id: String = ""

    @JsonProperty("Title") var title: String = ""

    @JsonProperty("SubTitle")
    var subTitle: String = ""

    @JsonProperty("Description")
    var description: String = ""

    @JsonProperty("Image")
    var imageUrl: String = ""
    var isbn: String = ""

}
